src/parser.js

```diff
@@ -221,16 +222,16 @@ function getFlowParent(defaultTransition, head, {node}) {
                 // the workflow and the flowStateGraph contains a single string. there are is parent.
                 return {};
             case 'array':
-                for (const index in pos)
-                    if (pos[index] === node) {
-                        const {stateName: posStateName} = destructString({
-                            node: pos,
-                            key: index
-                        }, {expectedToFind: true});
-                        if (posStateName === firstStateNameInFlow)
-                            return {foundFlowParent: true, foundNode: true, flowParentNode: pos, flowParentKey: index};
-                        return {foundNode: true};
-                    }
+                const element = _.find(pos, index => index === node);
```

Comment:
you can use array.prototype.find. most browsers or even all of them support this.